calico/etcddriver/driver.py

```diff
@@ -78,6 +80,8 @@
 
 # Threshold in seconds for detecting watcher tight looping on exception.
 REQ_TIGHT_LOOP_THRESH = 0.2
+# How often to log stats.
+STATS_LOG_INTERVAL = 10
```

Comment:
or maybe this will only trigger if we are under load, so we'll get away with it... for now, let's leave it then.